viewers/wicket/model/src/main/java/org/apache/isis/viewer/wicket/model/models/interaction/act/ActionInteractionWkt.java

```diff
@@ -143,7 +144,15 @@ protected ActionInteraction load() {
                         .get(getBookmarkedOwner()); //XXX make this nullToEmpty in OneToOneAssociation
                 val compositeValue =
                         ManagedObjects.nullToEmpty(prop.getElementType(), compositeValue0);
-                return ActionInteraction.start(compositeValue, memberId, where);
+
+                val action = valueTypeSpec.lookupFacet(ValueFacet.class)
```

Comment:
this is low-level stuff, should not be the responsibility of any viewer implementation